src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+
+        Ball ball = new Ball();
+
+        Random random = new Random();
```

Comment:
Make Random **class-level** variable. Now it is a **local** variable.

Reply:
Ok, but why this implementation is bad, these variables are used only in this method?

Reply:
dont write comments here, because mentors dont see them until the next review